### PR TITLE
Add move for xDSL and Fiber offers

### PR DESCRIPTION
**Pull request by Steffy29**
Add enpoints on API v6 for :
* eligibility test for copper and fiber lines
* connectivity searches for address and buildings
* moving offer which includes fiber offers
* getting contact owner (which is needed by move offer)

Reference: UXCT-193

### CODE REVIEW

**Comment by frenautvh** [comment]
i think some commits should be merged, it is unnecessary verbose (for example merge polling related commits into one)

**Comment by Steffy29** [comment]
@frenautvh commits are merged

File: src/api/pack/xdsl/pack-xdsl.v6.service.js [hunk 1]
```diff
@@ -77,5 +77,15 @@ angular.module('ovh-api-services').service('OvhApiPackXdslV6', ($resource, OvhAp
         packName: '@packName',
       },
     },
+    getContactOwner: {
+      method: 'GET',
+      url: '/pack/xdsl/:packName/contactOwner',
```
Comment (antleblanc) [inline src/api/pack/xdsl/pack-xdsl.v6.service.js:82]: API returns a type `pack.xdsl.ContactInfos` with the following properties:

```js
{
    address
    city
    country
    email
    firstname
    name
    organisation
    phone
    phoneCountry
    zip
}
```

Are you sure about the `transformResponse`?

File: src/api/pack/xdsl/move/pack-xdsl-move.v6.service.js [hunk 2]
```diff
@@ -38,5 +43,33 @@ angular.module('ovh-api-services').service('OvhApiPackXdslMoveV6', ($resource, P
     );
   };
 
+  move.pollOffers = function ($scope, opts) {
+    const url = ['/pack/xdsl/', opts.packName, '/addressMove/offers'].join('');
```
Comment (antleblanc) [inline src/api/pack/xdsl/move/pack-xdsl-move.v6.service.js:47]: You can use template strings:
```suggestion
    const url = `/pack/xdsl/${opts.packName}/addressMove/offers`;
```

File: src/api/connectivity/eligibility/connectivity-eligibility.v6.service.js [hunk 3]
```diff
@@ -1,8 +1,107 @@
...
-    method: 'POST',
-    isArray: false,
-    cache: OvhApiConnectivityEligibility.cache,
-  },
-}));
+angular.module('ovh-api-services').service('OvhApiConnectivityEligibilityV6', ($resource, OvhApiConnectivityEligibility, Poller) => {
+  const eligibility = $resource('/connectivity/eligibility', {
+  }, {
+    buildingDetails: {
+      url: '/connectivity/eligibility/search/buildingDetails',
```
Comment (antleblanc) [inline src/api/connectivity/eligibility/connectivity-eligibility.v6.service.js:5]: The `search` folder already exist. Why not keeping the same structure as mentionned into the README.md file?

See: https://github.com/ovh-ux/ovh-api-services/#structure

Comment (Steffy29) [reply src/api/connectivity/eligibility/connectivity-eligibility.v6.service.js:5]: Yes you're right, I moved this to the search folder.

File: src/api/pack/xdsl/pack-xdsl.v6.service.js (hunk 1 above)
Comment (Steffy29) [reply src/api/pack/xdsl/pack-xdsl.v6.service.js:82]: Yes I'm sure.

**Comment by ovh-cds** [comment]
CDS Report build#1571.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#1572.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#1573.0 ✘  
* 
  * build ✔ 
  * test ✘

**Pull request merged**